README.md

```diff
@@ -1,4 +1,29 @@
-# webapp-core [![Build Status](https://api.travis-ci.org/HerbertBodner/webapp-core.svg?branch=master)](https://travis-ci.org/HerbertBodner/webapp-core) [![AppVeyor Build](https://ci.appveyor.com/api/projects/status/github/HerbertBodner/webapp-core?branch=master&svg=true)](https://ci.appveyor.com/project/yshchohaleu/webapp-core)
-Reusable core features for an MVC Web application
+# webapp-core  
+Modules for common features for a .Net Core application
```

Comment:
Did you intentionally remove the phrase Web Application?

Reply:
Yes, I did, because the UoW module or Crud module are not only for web applications